src/etc/dhcp/dhclient-exit-hooks.d/static-dhclient-hook

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
```

Comment:
If you are creating a new hook, it should be named with order respect (`NN-myhook`). You are fixing the order issue with one hook and creating the same potential issue with another in the same commit. :)